packages/util-waiter/src/waiter.ts

```diff
@@ -0,0 +1,41 @@
+import { AbortController } from "@aws-sdk/abort-controller";
+
+export interface WaiterConfiguration {
+  /**
+   * The amount of time in seconds a user is willing to wait for a waiter to complete. This
+   * defaults to 300 (5 minutes).
```

Comment:
I don't recommend setting a default value for this. This is hands-down the most important waiter setting there is, and customers should tell us how long they're willing to wait.

Reply:
Oh this is actually old documentation. We dont set this anywhere. I'll change